src/vtab/csvtab.rs

```diff
@@ -1,6 +1,25 @@
-//! CSV Virtual Table.
+//! `feature = "csvtab"` CSV Virtual Table: https://www.sqlite.org/csv.html
```

Comment:
We are not using the official CSV extension...

Reply:
Hm, that's true, but it does say it's a port a few lines down, and the docs for this are better than our docs or the link to the source.

Would you prefer this link be placed with the spot that says it's a port?